core/src/test/resources/data_mutations_extended_json_annotation.txt

```diff
@@ -0,0 +1,52 @@
+#sequenced_samples: TCGA-AA-3664-01
+Hugo_Symbol	Entrez_Gene_Id	Center	Tumor_Sample_Barcode	Verification_Status	Validation_Status	Mutation_Status	Sequencer	Chromosome	Start_position	End_position	Variant_Classification	HGVSp_Short	MA:FImpact	MA:link.MSA	MA:link.PDB	NAMESPACE1:HEADER1	NAMESPACE1.HEADER2	NAMESPACE2.HEADER1	INVALID_NAMESPACE:HEADER1	INVALID_NAMESPACE.HEADER1	ASCN.ASCN_INTEGER_COPY_NUMBER	ASCN.ASCN_METHOD	ASCN.CCF_M_COPIES_UPPER	ASCN.CCF_M_COPIES	ASCN.CLONAL	ASCN.MINOR_COPY_NUMBER	ASCN.MUTANT_COPIES	ASCN.TOTAL_COPY_NUMBER
```

Comment:
Are the entries with colon delimiter meant to make sure we do not parse colon delimited strings as namespaces? (NAMESPACE1:HEADER1)